app/scripts/controllers/transactions/index.js

```diff
@@ -510,13 +510,17 @@ export default class TransactionController extends EventEmitter {
    * }
    * @param swapApprovalTransaction.type
    * @param swapApprovalTransaction.sourceTokenSymbol
+   * @param swapApprovalTransaction.estimatedBaseFee
    */
-  updateSwapApprovalTransaction(txId, { type, sourceTokenSymbol }) {
+  updateSwapApprovalTransaction(
+    txId,
+    { type, sourceTokenSymbol, estimatedBaseFee },
+  ) {
     if (!this._checkIfTxStatusIsUnapproved(txId)) {
       return;
     }
 
-    let swapApprovalTransaction = { type, sourceTokenSymbol };
+    let swapApprovalTransaction = { type, sourceTokenSymbol, estimatedBaseFee };
```

Comment:
estimatedBaseFee will actually be available in controllers code also I think as its value is taken from gasFeeEstimates.